pom.xml

```diff
@@ -0,0 +1,218 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.fcrepo</groupId>
+    <artifactId>fcrepo</artifactId>
+    <version>4.3.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>fcrepo-module-auth-webac</artifactId>
+  <name>Fedora Repository WebAC Authorization Module</name>
+  <description>Fedora Repository WebAC module</description>
+  <packaging>bundle</packaging>
+
+  <properties>
+    <osgi.import.packages>
+      org.fcrepo.auth.roles.common,
+      org.fcrepo.kernel.api.*,
+      org.fcrepo.http.commons.*,
+      org.fcrepo.kernel.modeshape.*,
+
+      javax.jcr.*,
+      javax.annotation,
+      javax.xml.*,
+      javax.inject,
+      javax.servlet.http,
+
+      com.google.common.*,
+      com.hp.hpl.jena.*,
+
+      org.apache.commons.io
+      org.modeshape.jcr.api.*,
+      org.springframework.beans.*,
+      org.springframework.stereotype,
+      org.w3c.dom.*,
```

Comment:
This and all of the explicit imports above should be removed for now. (leave the * on line 35)